source/common/res/features/close-reconcile-window/main.js

```diff
@@ -0,0 +1,21 @@
+(function poll() {
+  if (typeof ynabToolKit !== 'undefined' && ynabToolKit.actOnChangeInit === true) {
```

Comment:
@blargity No specific reason, I just copied from another feature, not shared/example.js. Many features use that line, but I'll know better in the future. Thanks.